Rubrik/Public/Get-RubrikSnapshotByDate.ps1

```diff
@@ -0,0 +1,34 @@
+﻿#Requires -Version 3
```

Comment:
Can you justify having a public function for this? Seems like adding a Date param to Get-RubrikSnapshot would be cleaner.